app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = \
```

Comment:
Code style is not that good. My advice is to format `sorted_variables` similarly to `marks`